client/src/deploy/sign.rs

```diff
@@ -0,0 +1,56 @@
+use std::{fs, io::Write};
+
+use clap::{App, ArgMatches, SubCommand};
+
+use super::creation_common;
+use crate::{command::ClientCommand, common};
+
+use casper_node::types::Deploy;
+
+pub struct SignDeploy;
+
+impl<'a, 'b> ClientCommand<'a, 'b> for SignDeploy {
+    const NAME: &'static str = "sign-deploy";
+    const ABOUT: &'static str = "Cryptographically signs a deploy. The signature \
+    is appended to existing approvals.";
+
+    fn build(display_order: usize) -> App<'a, 'b> {
+        let subcommand = SubCommand::with_name(Self::NAME)
+            .about(Self::ABOUT)
+            .display_order(display_order)
+            .arg(common::secret_key::arg(display_order))
+            .arg(creation_common::input::arg(display_order))
+            .arg(creation_common::output::arg(display_order));
+        creation_common::apply_common_creation_options(subcommand)
```

Comment:
Again, these should not be applied here.  They actually stop me even running the app with `-h` as the secret_key option is duplicated.